src/main/java/uk/gov/companieshouse/siccode/api/search/CombinedSicActivitiesStorageModel.java

```diff
@@ -104,48 +108,62 @@ public int hashCode() {
         result = prime * result
                 + ((activityDescriptionSearchField == null) ? 0 : activityDescriptionSearchField.hashCode());
         result = prime * result + ((id == null) ? 0 : id.hashCode());
-        result = prime * result + (isCompaniesHouseactivity ? 1231 : 1237);
+        result = prime * result + (companiesHouseactivity ? 1231 : 1237);
         result = prime * result + ((sicCode == null) ? 0 : sicCode.hashCode());
         result = prime * result + ((sicDescription == null) ? 0 : sicDescription.hashCode());
         return result;
     }
 
     @Override
     public boolean equals(Object obj) {
-        if (this == obj)
+        if (this == obj){
             return true;
-        if (obj == null)
+        }
+        if (obj == null){
             return false;
-        if (getClass() != obj.getClass())
+        }
+        if (getClass() != obj.getClass()){
             return false;
+        }
         CombinedSicActivitiesStorageModel other = (CombinedSicActivitiesStorageModel) obj;
         if (activityDescription == null) {
-            if (other.activityDescription != null)
+            if (other.activityDescription != null){
                 return false;
-        } else if (!activityDescription.equals(other.activityDescription))
+            }
+        } else if (!activityDescription.equals(other.activityDescription)){
             return false;
+        }
```

Comment:
You could consider Objects.equals here, replacing this whole set of lines with:
```java
if (!Objects.equals(activityDescription, other.activityDescription) {
    return false
}
```
See https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object-

Reply:
Tom - This is a generated equals method that we need in order to support the testing.

Personally I think it is safer to use a generated error method